README.md

```diff
@@ -1 +1,9 @@
 # genson-js
+
+**genson-js** is a user-friendly **JSON Schema** generator built in TypeScript/JavaScript.
```

Comment:
document available methods in readme

Reply:
yep, that whole publish to npm part is yet to be done (including ci, readme, npm stuff)